src/internal/modules/billing/routes/view.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const Joi = require('@hapi/joi');
+const controller = require('../controllers/view');
+const { scope } = require('../../../lib/constants');
+
+const allowedScopes = [scope.billing];
+
+module.exports = {
+  getBillsForLicence: {
+    method: 'GET',
+    path: '/licences/{licenceId}/bills',
```

Comment:
As this URL begins with `licences`, it should be located in the licences module. This is my understanding of how we organise our modules, anway